refarch-eai/src/test/resources/application-test.yml

```
@@ -4,10 +4,10 @@
       default: none
 
 logging:
-  include-application-group: false
-  level:
-    org:
-      springframework: INFO
+  include-application-group: false # disabled because application group is only relevant when deployed
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)